src/context/WalletProvider/NewWalletViews/wallets/mipd/MipdBody.tsx

```diff
@@ -211,7 +211,6 @@ export const MipdBody = ({ rdns, isLoading, error, setIsLoading, setError }: Mip
         loadingText={translate('common.pairing')}
         spinner={spinner}
         onClick={pairDevice}
-        data-test='wallet-pair-button'
```

Comment:
We're not testing this view right?

Reply:
Negative! e2e tests are effectively a thing of the past, wouldn't hurt to remove all `data-test` attributes across the app